sys-apps/smartmontools/smartmontools-9999.ebuild

```diff
@@ -68,13 +70,13 @@ src_install() {
 		newinitd "${FILESDIR}"/smartd-r1.rc smartd
 		newconfd "${FILESDIR}"/smartd.confd smartd
 
-		keepdir ${db_path}
+		keepdir "${db_path}"
 		if use update_drivedb ; then
 			# Move drivedb.h file out of PM's sight (bug #575292)
-			mv "${ED}"${db_path}/drivedb.h "${T}" || die
+			mv "${ED}${db_path}/drivedb.h" "${T}" || die
 
 			exeinto /etc/cron.monthly
-			doexe "${FILESDIR}"/${PN}-update-drivedb
+			doexe "${FILESDIR}/${UPDATE_DRIVEDB}"
```

Comment:
Why are you trying to install this cron script with different name but did not either

- change the file's name in FILESDIR

or

- use `newexe` to rename the file on installation
Please fix!